### PR TITLE
Rf bugfix

### PR DESCRIPTION
**Pull request by rssfrncs**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -180,33 +162,41 @@ function App() {
...
-  update({
-    type: "highest rated movies set",
-    hRMPayload: sortArray(state.result.Search),
-  });
+  const sortedMovies = React.useMemo(() => {
+    return state.result
+      ? state.result.Search.slice(0).sort((a, b) => {
+          return parseInt(a.imdbRating) - parseInt(b.imdbRating);
+        })
+      : [];
```
Comment (lwswr) [inline src/App.tsx:196]: Why is there an empty array rather than null here?

File: src/App.tsx [hunk 2]
```diff
@@ -180,33 +162,41 @@ function App() {
   useEffect(() => {
     async function callToAPIs() {
       try {
-        const [movieResponse, selMovieResponse] = await Promise.all([
+        const [movieResponse] = await Promise.all([
           getMovies(state.search, state.page, mediaType),
           getSelected(selectedID),
```
Comment (lwswr) [inline src/App.tsx:167]: Can I remove the getSelected func call from here if I'm calling it in the other useEffect?

Comment (rssfrncs) [reply src/App.tsx:167]: whoops yeah definitely remove it

File: src/App.tsx (hunk 1 above)
Comment (rssfrncs) [reply src/App.tsx:196]: i quite like to return the same type if possible so an empty array is still `Movie[]` as opposed to `Movie[] | null`.

**Pull request merged**